src/main/kotlin/it/unibo/protelis/web/execution/simulated/NewSimulatedProtelisEngine.kt

```diff
@@ -0,0 +1,82 @@
+package it.unibo.protelis.web.execution.simulated
+
+import it.unibo.alchemist.core.implementations.Engine
+import it.unibo.alchemist.core.interfaces.Simulation
+import it.unibo.alchemist.core.interfaces.Status
+import it.unibo.alchemist.core.interfaces.waitForAndCheck
+import it.unibo.alchemist.loader.YamlLoader
+import it.unibo.alchemist.model.implementations.positions.Euclidean2DPosition
+import it.unibo.alchemist.model.implementations.times.DoubleTime
+import it.unibo.protelis.web.execution.ProtelisObserver
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+class NewSimulatedProtelisEngine {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [UndocumentedPublicClass - [NewSimulatedProtelisEngine] at /src/src/main/kotlin/it/unibo/protelis/web/execution/simulated/NewSimulatedProtelisEngine.kt:16:1](https://app.codacy.com/manual/NiccoMlt/Protelis-Web/pullRequest?prid=5040950)